README.md

```diff
@@ -1,2 +1,37 @@
-# README
-This will be edited later.
+# Ruby on Rails Facebook application
+
+This is the Final Poject in The Odin Project’s Ruby on Rails Curriculum. For more information click --> [Facebook-Odin]
```

Comment:
Mentioning that this is a project from the Odin project is not necessary, it would look better if you simply stated that it is a clone of Facebook.
You can mention:
- The technology stack you used and the versions of the technologies and why
- Inspiration for project
- Approach to project
- Whatever else you feel like 

[OPTIONAL]